samples/CloudNative.CloudEvents.AspNetCoreSample/CloudNative.CloudEvents.AspNetCoreSample.http

```diff
@@ -0,0 +1,12 @@
+@HostAddress = https://localhost:5001
+
+POST {{HostAddress}}/api/events/receive
+Content-Type: application/json
+CE-SpecVersion: 1.0
+CE-Type: "com.example.myevent"
+CE-Source: "urn:example-com:mysource:abc"
+CE-Id: "c457b7c5-c038-4be9-98b9-938cb64a4fbf"
```

Comment:
In an ideal world, I wouldn't hardcode the GUID here and would instead use dynamic variable support provided by `.http` files. However, it appears that each editor recognizes a different dynamic variable for generating a GUID.

VS Code and VS respect `{{$guid}}` and Rider respects `{{$random.uuid}}` so there is no universal value to include here. Opted to hard code for now.